network-api/networkapi/wagtailpages/utils.py

```diff
@@ -114,39 +117,50 @@ def get_content_related_by_tag(page, result_count=3):
     on its `.tags` content. If it has tags.
     """
 
-    tagged_post_types = (
+    if hasattr(page.specific, 'tags') is False:
+        return list()
+
+    page_models_with_tags = (
         apps.get_model('wagtailpages', 'BlogPage'),
-        # Add more models here as we add tags to more types
+        #
+        # Uncomment the following class(es) to add them to the
+        # set of classes to use for tag matching:
+        #
+        # apps.get_model('wagtailpages', 'BanneredCampaignPage'),
+        # apps.get_model('wagtailpages', 'CampaignPage'),
+        # apps.get_model('wagtailpages', 'OpportunityPage'),
```

Comment:
Are we uncommenting these out in follow-up PRs?

Reply:
yeah, this issue was technically for laying the ground works, we don't have any comps at all for what things should actually look like on (bannered) campaign/opportunity pages yet.